lmdeploy/pytorch/engine/logits_process.py

```diff
@@ -276,6 +276,11 @@ def __random_sampling(scores: torch.Tensor, indices: torch.LongTensor):
                 scores = _filter_topp_sorted(scores, top_p)
 
             softmax_scores = scores.softmax(1)
+
+            max_topk = sampling_inputs.max_top_k
```

Comment:
`max_top_k` can be 0, which means `top_k` sampling is skipped.

Reply:
top_k==0 is equal to top_k=vocab_size ?

Reply:
Yes. Have we updated it to `vocab_size` when users' inputting `top_k` is 0?

Reply:
Never